Bolts/Common/BFTask.m

```diff
@@ -172,6 +172,75 @@ + (instancetype)taskForCompletionOfAllTasksWithResults:(NSArray<BFTask *> *)task
     }];
 }
 
++ (instancetype)taskForCompletionOfAnyTask:(NSArray<BFTask *> *)tasks
+{
+    __block int32_t total = (int32_t)tasks.count;
+    if (total == 0) {
+        return [self taskWithResult:nil];
+    }
+    
+    __block dispatch_once_t token = 0;
```

Comment:
dispatch_once_t must point to a variable stored in a global or static scope. The result of using a predicate like this is actually `undefined`, according to documentation:
https://developer.apple.com/library/ios/documentation/Performance/Reference/GCD_libdispatch_Ref/

You can use `OSAtomicTestAndSet` for this with a boolean.